src/bundle/Resources/public/js/alloyeditor/src/toolbars/ez-add.js

```diff
@@ -7,34 +7,60 @@ export default class EzToolbarAdd extends AlloyEditor.Toolbars.add {
         return 'ezadd';
     }
 
+    /**
+     * Checks if anything was selected in editor.
+     * There may be situation when editor has focus but none of its elements.
+     *
+     * @returns {Boolean}
+     */
+    checkIsNothingSelected() {
+        const { selectionData } = this.props;
+
+        if (!selectionData || !selectionData.region) {
+            return false;
+        }
+
+        const { top, bottom, left, right } = selectionData.region;
+
+        return !top && !bottom && !left && !right;
+    }
+
     /**
      * Lifecycle. Renders the UI of the button.
      *
      * @method render
-     * @return {Object} The content which should be rendered.
+     * @returns {Object} The content which should be rendered.
      */
     render() {
         const selectionData = this.props.selectionData;
+        const nothingSelected = this.checkIsNothingSelected();
```

Comment:
Nitpick: this variable is not required. The function itself is pretty self-explanatory, so I would avoid declaring this variable.